src/vivarium_gates_mncnh/constants/data_values.py

```diff
@@ -421,6 +423,12 @@ class __Interventions(NamedTuple):
 AZITHROMYCIN_RELATIVE_RISK_DISTRIBUTION = get_lognorm_from_quantiles(1.54, 1.30, 1.82)
 MISOPROSTOL_RELATIVE_RISK_DISTRIBUTION = get_lognorm_from_quantiles(0.61, 0.50, 0.74)
 
+# Effects of IV iron intervention
+IV_IRON_HEMOGLOBIN_EFFECT_SIZE = {
+    "Ethiopia": get_norm(20.2, (21.5 - 18.9) / (2 * 1.96)),
```

Comment:
Would be great to include the documentation link from the description of this PR as a comment here, so people can trace it back easier in the future!